frontend/src/pages/GettingStarted.tsx

```diff
@@ -67,8 +71,9 @@ This section contains demo and tutorial pipelines.
 
 **Tutorials** - Learn pipeline concepts by following a tutorial.
 
-  * [Data passing in python components](${data}) - Shows how to pass data between python components. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components)
-  * [DSL - Control structures](${control}) - Shows how to use conditional execution and exit handlers. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures)
+* [V2 lightweight Python components](${v2python}) - Shows different component input and output options for KFP v2 components. [source code](https://github.com/kubeflow/pipelines/blob/master/samples/v2/lightweight_python_functions_v2_pipeline/lightweight_python_functions_v2_pipeline.py)
```

Comment:
nit: V2 Python function based components

Refer to title in https://www.kubeflow.org/docs/components/pipelines/sdk/v2/python-function-components/.
lightweight python component is a slightly confusing term, because what is heavyweight? aren't all components used in python DSL?

Reply:
Updated description as suggested. I agree that the `lightweight` adj is confusing. This is taken from the pipeline tutorial sample names, so we should update them as well. But it is out-of-scope for this PR:

![pipelinelist](https://user-images.githubusercontent.com/37026441/129519586-6530544e-cf0e-4189-aa03-30c4c6f87370.png)